website/source/docs/telemetry/overview.html.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: "docs"
+page_title: "Overview"
+sidebar_current: "docs-telemetry-overview"
+description: |-
+  Overview of runtime metrics available in Nomad along with monitoring and
+  alerting.
+---
+
+# Telemetry Overview
+
+The Nomad client and server agents collect a wide range of runtime metrics
+related to the performance of the system. On the server side, leaders and
+followers have metrics in common as well as metrics that are specific to their
+roles. Clients have separate metrics for the host metrics and for
+allocations/tasks, both of which have to be [explicitly
+enabled][telemetry-stanza]. There are also runtime metrics that are common to
+all servers and clients.
+
+Metrics are aggregated on a ten second interval and are retained for one minute.
```

Comment:
are these default configurations or hard-coded values? 
if default configs, I would explicitly say "By default" and link to the configuration page.

Reply:
Okay there may be a mistake here because it looks like the default collection interval is 1s

https://www.nomadproject.io/docs/configuration/telemetry.html#collection_interval

Reply:
Good question. Going to dig into this one for a bit.

Reply:
Thinking about taking this line out and just stating that the default collection interval for nomad is 1s (and add a link to the configuration page detailing that).